com.varnamproject.Varnam.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
You should add `cleanup` array to remove files such as static libraries and development headers.

Reply:
Thank you, after adding it the package got reduced by over 15MB ! I'm not that familiar with C, so thought `.a` files were needed

Reply:
Found after the build, there's a `lib/debug` folder with some ruby related files in it. Is this folder necessary or shall I add it to cleanup ?

Reply:
`lib/debug` will be automatically split to `Debug` extension, it should be kept as is. As for recommended list, it usually differs between applications. You can search existing manifests with GitHub search to see what others cleaned up from Ruby.